src/main/java/net/glowstone/entity/objects/GlowArmorStand.java

```diff
@@ -0,0 +1,491 @@
+package net.glowstone.entity.objects;
+
+import com.flowpowered.networking.Message;
+import java.util.Arrays;
+import java.util.List;
+import net.glowstone.EventFactory;
+import net.glowstone.GlowWorld;
+import net.glowstone.entity.GlowLivingEntity;
+import net.glowstone.entity.GlowPlayer;
+import net.glowstone.entity.meta.MetadataIndex;
+import net.glowstone.net.message.play.entity.EntityEquipmentMessage;
+import net.glowstone.net.message.play.entity.EntityMetadataMessage;
+import net.glowstone.net.message.play.entity.SpawnObjectMessage;
+import net.glowstone.net.message.play.player.InteractEntityMessage;
+import net.glowstone.util.Position;
+import org.bukkit.Effect;
+import org.bukkit.GameMode;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.entity.ArmorStand;
+import org.bukkit.entity.Arrow;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.Projectile;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.event.entity.EntityDamageEvent;
+import org.bukkit.event.player.PlayerArmorStandManipulateEvent;
+import org.bukkit.inventory.EquipmentSlot;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.util.EulerAngle;
+
+public class GlowArmorStand extends GlowLivingEntity implements ArmorStand {
+
+    private final ItemStack[] equipment = new ItemStack[5];
+    private final boolean[] changedEquip = new boolean[5];
+    private final EulerAngle[] pose = new EulerAngle[6];
+
+    private boolean isMarker = false;
+    private boolean isVisible = true;
+    private boolean isSmall = false;
+    private boolean hasBasePlate = true;
+    private boolean hasGravity = true;
+    private boolean hasArms = false;
+
+    public GlowArmorStand(Location location) {
+        super(location);
+        setMaxHealth(2);
+        setHealth(2);
```

Comment:
@mastercoms When we pull #197 first then here can be used the new super constructor for GlowLivingEntity